src/frontends/tensorflow_common/src/place.cpp

```diff
@@ -40,19 +40,23 @@ const std::map<std::string, std::vector<std::shared_ptr<InPortPlace>>>& OpPlace:
 }
 
 std::shared_ptr<InPortPlace> OpPlace::get_input_port_tf(const std::string& inputName, int inputPortIndex) const {
-    FRONT_END_GENERAL_CHECK(inputPortIndex <= m_input_ports.at(inputName).size(), "inputPortIndex is out of bounds.");
-    return m_input_ports.at(inputName)[inputPortIndex];
+    FRONT_END_GENERAL_CHECK(inputPortIndex >= 0, "inputPortIndex is negative.");
+    size_t input_port_index = static_cast<size_t>(inputPortIndex);
+    FRONT_END_GENERAL_CHECK(input_port_index <= m_input_ports.at(inputName).size(), "inputPortIndex is out of bounds.");
+    return m_input_ports.at(inputName)[input_port_index];
 }
 
 std::shared_ptr<DecoderBase> OpPlace::get_decoder() const {
     return m_op_decoder;
 }
 
 void OpPlace::add_out_port(const std::shared_ptr<OutPortPlace>& output, int idx) {
```

Comment:
should not it be `size_t idx`? do we have cases when `idx` is really negative?

Reply:
Please take a look at common `Place` class where we use `int` type for indices: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/common/include/openvino/frontend/place.hpp#L122

That is why other custom functions also have `int` type to be aligned with it:)

Reply:
maybe we need to change base class? do we need to have int as `idx`? is something as `-1` is used as a special value?

Reply:
I don't know. If I start updating the base class. it will lead to the update in all frontends. Absolutely this is different story:)